architecture/additional_concepts/storage.adoc

```diff
@@ -96,7 +96,7 @@ volume and mounts that volume for a pod. For those volumes that support multiple
 access modes, you must specify which mode applies when you use the claim as
 a volume in a pod.
 
-Once you have a claim and that claim is bound, the bound PV belongs to you 
+Once you have a claim and that claim is bound, the bound PV belongs to you
```

Comment:
s/Once/After

Reply:
Yep, that's a good catch. Fixed! Thanks @kalexand-rh